src/frontends/onnx/frontend/src/onnx_framework_node.cpp

```diff
@@ -26,5 +26,24 @@ std::shared_ptr<Node> ONNXSubgraphFrameworkNode::clone_with_new_inputs(const Out
     return std::make_shared<ONNXSubgraphFrameworkNode>(m_node, m_functions, inputs);
 }
 
+std::shared_ptr<Node> NotSupportedONNXNode::clone_with_new_inputs(const OutputVector& inputs) const {
+    auto attrs = get_attrs();
+    std::string error_message = attrs[failed_conversion_key];
+    return std::make_shared<NotSupportedONNXNode>(inputs,
+                                                  get_output_size(),
+                                                  attrs.get_opset_name(),
+                                                  attrs.get_type_name(),
+                                                  error_message);
+}
+
+bool NotSupportedONNXNode::visit_attributes(AttributeVisitor& visitor) {
+    auto attrs = get_attrs();
+    auto domain = attrs.get_opset_name();
+    auto op_type = attrs.get_type_name();
```

Comment:
Use `const auto&`

Reply:
done for `attr`, can not do for `domain` and `op_type` because `on_adapter` doesn't match a type for such a case